### PR TITLE
Update img link tenant database management

### PR DESCRIPTION
**Pull request by adela-bytebase**
There are some link URLs look like https://lh5.googleusercontent.com/rMDiat5aaHH5Y4W11dyJJ6BaQFBKpgWpN1fe7I6KDa5Lr9KFl8IBvkDxBTN9PfWnUciyFsDPrp4MM2IMXKBbEunyWjfR4OoTA9LwcBci_DnTvrqAcEv17toC9P5V7sztyyhsHtOI , updated them into static/docs-assets/xxxx.png

### CODE REVIEW

**Comment by boojack** [comment]
Maybe you need to upload those images in this PR.

File: docs/features/tenant-database-mamangement.md [hunk 1]
```diff
@@ -13,15 +13,16 @@ Let's take the hospital example to follow the steps below.
 
 ## **Labels**
 
-A label is a key-value pair that helps you identify the tenant for a database. The supported label keys are location and tenant currently. Workspace owner needs to predefine label values (tenant names) in the Workspace Label tab.
+A label is a key-value pair that helps you identify the tenant for a database. The supported label keys are location and tenant currently. Workspace owner needs to predefine label values (tenant names) in the Workspace Label tab.dddd
```
Comment (raytian03) [inline docs/features/tenant-database-mamangement.md:16]: ddd should be removed.

**Review by raytian03 — CHANGES_REQUESTED** [review]
Please upload the images

File: docs/features/tenant-database-mamangement.md [hunk 2]
```diff
@@ -13,15 +13,16 @@ Let's take the hospital example to follow the steps below.
...
-![](https://lh5.googleusercontent.com/rMDiat5aaHH5Y4W11dyJJ6BaQFBKpgWpN1fe7I6KDa5Lr9KFl8IBvkDxBTN9PfWnUciyFsDPrp4MM2IMXKBbEunyWjfR4OoTA9LwcBci_DnTvrqAcEv17toC9P5V7sztyyhsHtOI)
+![Tenant Labels](/static/docs-assets/tntdbmngmt-tenant-labels.png)
 
 ## **Projects in Tenant Mode**
 
 Project can be created in Tenant Mode which supports tenant database management.
 
-![](https://lh6.googleusercontent.com/eCSBofNQlemoRd1hibP3uFVvxR_KBXx9AxTETaZI5DUhKSxSoZTJYpE96dOrR-dogpQMe01gjrzHyEAaz420fOVtT05Z7BHcA1ZSw19ZF0MS4shA4-y7Sc7TfSis3rRA8CKoXsU5)
+![Create Project in Tenant Mode](/static/docs-assets/tntdbmngmt-create-project-in-tenant-mode.png)
+
```
Comment (boojack) [inline docs/features/tenant-database-mamangement.md:25]:
```suggestion
```

File: docs/features/tenant-database-mamangement.md [hunk 3]
```diff
@@ -35,44 +36,45 @@ You need to define deployment configuration for a project in Tenant Mode before
...
-![](https://lh5.googleusercontent.com/fZzLjuIq3psIv67snNyZG3cE1KPVolLOG75QYWEZTZimvwCKK3qrscNoH8d43RU99E4RYfNaK9Cs9NhxjeDvZFY_sd75K_6-1xPWzR8wu_gKs0ik0_okKPaOmh69LqqXb9jnEl3I)
+![Tenant Databases](/static/docs-assets/tntdbmngmt-tenant-databases.png)
 
 ## **Schema Update for tenant databases**
 
 There is a preview dialog showing the plan of deployment before the schema is altered.
 
-![](https://lh5.googleusercontent.com/ZtI6JsVO19yJTuPDgZWyJrC8ITc9xc-51P198xoY7JaswEEfZ-NwgiypJhJAPyG_585MQGKqOabiAhEqvhxFxqJkJFEW8aSvrj3u4ZJr1X33z6QG5pIdHta6IP670Roi7HElQgSt)
+![Alter Schema](/static/docs-assets/tntdbmngmt-alter-schema.png)
+
```
Comment (boojack) [inline docs/features/tenant-database-mamangement.md:56]:
```suggestion
```

**Review by boojack — CHANGES_REQUESTED** [review]
extra blank line should be removed.

**Review by raytian03 — DISMISSED** [review]
LGTM

File: docs/features/tenant-database-mamangement.md [hunk 4]
```diff
@@ -35,44 +35,44 @@ You need to define deployment configuration for a project in Tenant Mode before
...
 
-![](https://lh4.googleusercontent.com/qhiFW7AVlgR1qf6kn8QNtwBEBELrd0Wz3HiTWVbN4uUz5ZzfH_KbSGdUL3LCSVQYGt8urMW4BVn9WuMugzdouN-zlnoxPOnGSuPbfT5xPWQX5wNrBU8SsVN4UdqTcGyn-JmNF5Wv)
+![Deployment Config](/static/docs-assets/tntdbmngmt-deployment-config.png)
 
 ## **Databases with tenant labels**
 
 You can create databases with labels in tenant mode projects. The labels come from predefined label keys and values in the Workspace. You can also transfer an existing database from another project if its database name and schema follows the existing tenant pattern.
 
-![](https://lh5.googleusercontent.com/np56A-QL4l2jaCUbjVq7SehWbzyAK9gicnYIUAt_qhOd59galXdtWrfCSa2tereGS5oOyBTGWgGq5kSQ_iRd_IUi9j75A_hy-t35M6qezSy3Kvze2LB16fidknhPOgQ9nOuJcduC)
+![Create Database with Tenant Label](/static/docs-assets/create-database-with-tenant-label.png)
```
Comment (boojack) [inline docs/features/tenant-database-mamangement.md:44]: This image is not found.
```suggestion
![Create Database with Tenant Label](/static/docs-assets/tntdbmngmt-create-database-with-tenant-label.png)
```

**Review by boojack — APPROVED** [review]
LGTM

**Comment by tianzhou** [comment]
@adela-bytebase There are conflicts you need to resolve.

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**